core/server/master/src/main/java/alluxio/master/backup/BackupLeaderRole.java

```diff
@@ -250,16 +253,15 @@ private void activateWorkerConnection(Connection workerConnection) {
    */
   private void scheduleLocalBackup(BackupPRequest request) {
     mLocalBackupFuture = mExecutorService.submit(() -> {
-      try (LockResource lr = new LockResource(mStatePauseLock)) {
-        try {
-          mBackupTracker.updateState(BackupState.Running);
-          AlluxioURI backupUri = takeBackup(request, mBackupTracker.getEntryCounter());
-          mBackupTracker.updateBackupUri(backupUri);
-          mBackupTracker.updateState(BackupState.Completed);
-        } catch (IOException e) {
-          mBackupTracker.updateError(
-              new BackupException(String.format("Local backup failed: %s", e.getMessage()), e));
-        }
+      try (LockResource stateLockResource =
+          new LockResource(mStatePauseLock, mStateLockTimeout, TimeUnit.MILLISECONDS)) {
```

Comment:
If this times out, will the backup not be scheduled until the next time, or will there be some sort of retry?

Reply:
It's already scheduled during this call. However, the backup-tracker will be completed in the catch statement if it times out.

Reply:
Is there a retry outside of this? If the backup fails to lock, I feel like there should be retries.

Reply:
No retries before, and didn't need to add more retries, because `tryLock` had the exact call implemented.